From release 18:
    * The compatibility launcher is no longer isolated in its own
      network namespace, since the SELinux APIs require access to the
      root network namespace.  If you package the launcher with SELinux
      disabled, you can get back the old behavior by using
      `PrivateNetwork=true` in your dbus service file.

--

diff --git a/src/units/system/dbus-broker.service.in b/src/units/system/dbus-broker.service.in
index 4155630..97f1655 100644
--- a/src/units/system/dbus-broker.service.in
+++ b/src/units/system/dbus-broker.service.in
@@ -14,6 +14,7 @@ LimitNOFILE=16384
 ProtectSystem=full
 PrivateTmp=true
 PrivateDevices=true
+PrivateNetwork=true
 ExecStart=@bindir@/dbus-broker-launch --scope system --audit
 ExecReload=@bindir@/busctl call org.freedesktop.DBus /org/freedesktop/DBus org.freedesktop.DBus ReloadConfig
